HPS/Heartland/view/frontend/templates/cards_list.phtml

```diff
@@ -0,0 +1,99 @@
+
+<script>
+    var <?php
+
+            $code = \HPS\Heartland\Model\Payment::CODE;
+            /* @noEscape */ echo $code; ?>;
+    require([
+                'jquery'
+            ],
+
+            /*
+             *  Heartland payment method model
+             *
+             *  @category    HPS
+             *  @package     HPS_Heartland
+             *  @author      Charlie Simmons <charles.simmons@e-hps.com>
+             *  @copyright   Heartland (http://heartland.us)
+             *  @license     https://github.com/hps/heartland-magento2-extension/blob/master/LICENSE.md
+             */
+
+            /*browser:true*/
+            /*global define*/
+            function ($) {
+                'use strict';
+                /**
+                 * Get the public key from HPS/Heartland/Controller/Hss/Pubkey.php as url configured based on HPS/Heartland/etc/frontend/routes.xml
+                 * if there is any form of error we disable the payment form
+                 *
+                 */
+                $(document).ready(function ($) {
+                    $("#iframes").fadeOut();
+                    <?php /* @noEscape */ echo $code; ?> = {
+
+
+                        deleteCard: function (id,e) {
+                            if(confirm('Are you sure you want to delete the stored card?')){
+                                $.ajax({
+                                    url: "<?php echo $block->getBaseUrl() ?>/heartland/creditcard/delete?t="+id
+                                    , success: function (data) {
+                                        e.remove();
+                                        alert('Your saved card has been removed from our records ');
+                                        location.reload();
+                                    }
+                                });
+                            }
+
+                        }
+
+
+
+
+                    }
+                });
+
+
+            }
+    );
+
+
+</script>
+<table id="SavedCardsTable">
```

Comment:
This table is required for design purpose, the page display "You have no saved cards" in tr tag when there are no cards are stored.